### PR TITLE
Correct CSS `display`

### PR DESCRIPTION
**Pull request by ExE-Boss**
This corrects the browser compatibility table data for the [`display` CSS property](https://developer.mozilla.org/docs/Web/CSS/display).

I’ve also added `display: ‑moz‑*` values because of [bug&nbsp;1255315](https://bugzil.la/1255315 "NEW - Can't find the meaning of &quot;display: -moz-box&quot; style attribute"), which will eventually be hidden from web content by [~~bug&nbsp;1288572~~](https://bugzil.la/1288572 "RESOLVED FIXED - Hide -moz-prefixed display values from web content") and [bug&nbsp;879275](https://bugzilla.mozilla.org/show_bug.cgi?id=879275 "NEW - Consider turning off -moz-box display types in untrusted stylesheets"), similarly to what was done with `‑moz‑border‑*‑colors` and `:‑moz‑system‑metric(…)`.

review?(@Elchi3, @ddbeck)

### CODE REVIEW

File: css/properties/display.json [hunk 1]
```diff
@@ -39,10 +36,13 @@
...
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "1"
             },
             "samsunginternet_android": {
               "version_added": true
+            },
+            "webview_android": {
+              "version_added": "1"
```
Comment (connorshea) [inline css/properties/display.json]: is there a version 1 for webview_android? There's no browser JSON for it so I'm not sure what the versioning system is like.

Comment (ExE-Boss) [reply css/properties/display.json]: needinfo?(@jpmedley)

Comment (jpmedley) [reply css/properties/display.json]: For something that old, I'd just put `true`. I don't have good information on versions that far back. 

Webview follows Chrome numbering.

Comment (ExE-Boss) [reply css/properties/display.json]: Well, Chrome for Desktop says `1` for that.

File: css/properties/display.json [hunk 2]
```diff
@@ -51,30 +51,77 @@
...
+              },
+              "samsunginternet_android": {
                 "version_added": null
               },
+              "webview_android": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (Loirooriol) [inline css/properties/display.json]: Multi-keyword values are at risk, and may be dropped during the CR period, just like `run-in`, which has `"experimental": true`.

File: css/properties/display.json [hunk 3]
```diff
@@ -525,46 +702,45 @@
         },
         "subgrid": {
```
Comment (Loirooriol) [inline css/properties/display.json]: `display: subgrid` was removed in https://github.com/w3c/csswg-drafts/commit/5a98e20279e49ae61680addc03726b0a421e7df8 in favor of per-axis subgrids.

Comment (ExE-Boss) [reply css/properties/display.json]: Done in 10c92d64ca3c3eaab3287ea1ca6de97527048aed

**Comment by ExE-Boss** [comment]
review?(@Elchi3)

**Comment by ExE-Boss** [comment]
review?(@ddbeck)

File: css/properties/display.json [hunk 4]
```diff
@@ -871,6 +1071,279 @@
...
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": true,
+                  "notes": "Scheduled for removal (See <a href='https://bugzil.la/879275>bug 879275)"
```
Comment (Elchi3) [inline css/properties/display.json]: Notes should end with a full stop and the link should have proper markup.
`"Scheduled for removal (see <a href='https://bugzil.la/879275'>bug 879275</a>)."`

File: css/properties/display.json [hunk 5]
```diff
@@ -871,6 +1071,279 @@
...
+                },
+                "firefox_android": {
+                  "version_added": true,
+                  "notes": "Scheduled for removal (See <a href='https://bugzil.la/879275>bug 879275)"
```
Comment (Elchi3) [inline css/properties/display.json]: Same here.

File: css/properties/display.json [hunk 6]
```diff
@@ -871,6 +1071,279 @@
...
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": true,
+                  "version_removed": "62",
+                  "notes": "See <a href='https://bugzil.la/1288572>bug 1288572"
```
Comment (Elchi3) [inline css/properties/display.json]: The bug is fixed. I don't think we need this note. (also, the markup is wrong/incomplete)

File: css/properties/display.json [hunk 7]
```diff
@@ -871,6 +1071,279 @@
...
+                },
+                "firefox_android": {
+                  "version_added": true,
+                  "version_removed": "62",
+                  "notes": "See <a href='https://bugzil.la/1288572>bug 1288572"
```
Comment (Elchi3) [inline css/properties/display.json]: ditto

File: css/properties/display.json [hunk 8]
```diff
@@ -871,6 +1071,279 @@
...
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": true,
+                  "version_removed": "62",
+                  "notes": "See <a href='https://bugzil.la/1288572>bug 1288572"
```
Comment (Elchi3) [inline css/properties/display.json]: ditto

File: css/properties/display.json [hunk 9]
```diff
@@ -871,6 +1071,279 @@
...
+                },
+                "firefox_android": {
+                  "version_added": true,
+                  "version_removed": "62",
+                  "notes": "See <a href='https://bugzil.la/1288572>bug 1288572"
```
Comment (Elchi3) [inline css/properties/display.json]: ditto

File: css/properties/display.json [hunk 10]
```diff
@@ -871,6 +1071,279 @@
...
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": true,
+                  "version_removed": "62",
+                  "notes": "See <a href='https://bugzil.la/1288572>bug 1288572"
```
Comment (Elchi3) [inline css/properties/display.json]: ditto

File: css/properties/display.json [hunk 11]
```diff
@@ -871,6 +1071,279 @@
...
+                },
+                "firefox_android": {
+                  "version_added": true,
+                  "version_removed": "62",
+                  "notes": "See <a href='https://bugzil.la/1288572>bug 1288572"
```
Comment (Elchi3) [inline css/properties/display.json]: ditto

File: css/properties/display.json [hunk 12]
```diff
@@ -871,6 +1071,279 @@
...
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": true,
+                  "version_removed": "62",
+                  "notes": "See <a href='https://bugzil.la/1288572>bug 1288572"
```
Comment (Elchi3) [inline css/properties/display.json]: ditto

File: css/properties/display.json [hunk 13]
```diff
@@ -871,6 +1071,279 @@
...
+                },
+                "firefox_android": {
+                  "version_added": true,
+                  "version_removed": "62",
+                  "notes": "See <a href='https://bugzil.la/1288572>bug 1288572"
```
Comment (Elchi3) [inline css/properties/display.json]: ditto

**Pull request merged**